### DISCUSSION TITLE
First person / "serverside" mode

### DISCUSSION THREAD

**Discussion by Attano** [Sixze/ALS-Refactored#55] [Ideas]
Would you(the author) be interested in adding a "serverside mode" toggle? By that I mean a mode in which ALS functions only for the purpose of providing third person animations for other clients, but not the owner. Out of the box it's not suitable for this because it's adding rotation to the character, which affects the first person view.

**Comment by Sixze** [comment]
For this purpose, you can disable mesh rendering on the owning client using the `bOwnerNoSee` setting. 

![image](https://user-images.githubusercontent.com/16257871/149175353-15c0556d-0214-4c9c-895e-da1c1ce7855c.png)

I don't quite understand what you mean about rotation. The rotation of the camera does not depend on the rotation of the character, so there should be no problem.
